src/refda/adm/ietf_amm.c

```diff
@@ -48,7 +48,8 @@ int refda_adm_ietf_amm_init(refda_agent_t *agent)
     REFDA_AGENT_LOCK(agent, REFDA_AGENT_ERR_LOCK_FAILED);
 
     cace_amm_obj_ns_t *adm =
-        cace_amm_obj_store_add_ns(&(agent->objs), "ietf-amm", "2024-07-03", true, REFDA_ADM_IETF_AMM_ENUM_ADM);
+        cace_amm_obj_store_add_ns(&(agent->objs), cace_amm_idseg_ref_withenum("ietf", REFDA_ADM_IETF_ENUM),
```

Comment:
The CAMP generated C source (like this) will need to be self contained, so references to external things like this can have symbolic comments but the enum values themselves will need to be literal and not preprocessor macros.